draft-ietf-rats-eat-measured-component.md

```diff
@@ -324,6 +325,9 @@ Note that similar security considerations may apply when the Measured Component
 The Component Name and Component Version can give an attacker detailed information about the software running on a device and its configuration settings.
 This information could offer an attacker valuable insights.
 
+Any textual fields (e.g., Component Name and Version) that are stored in a file, inserted into a database or displayed to humans must be properly sanitised to prevent attacks and undesirable behaviour.
```

Comment:
```suggestion
Any textual fields (e.g., Component Name and Version) that are stored in a file, inserted into a database, or displayed to humans must be properly sanitized to prevent attacks and undesirable behavior.
```